---
layout: assay
title: Exome sequencing
tags:
    - assay
browser_text: Cell lines with exome-seq data
browser_link: /lines?Assays[]=Exome-seq
---

Exome sequencing is performed on all HipSci iPS cell lines selected for banking
after passing QC. Sequencing and primary analysis  are performed at the
Wellcome Trust Sanger Institue. The experimental protocols are available on the
[SOP pages]({% post_url 1111-11-11-sop %}).

##Getting the data

The [cell lines and data browser]({{'/lines/#' | append: page.browser_link | prepend: site.baseurl}})
contains links to HipSci's Exome seq data for any cell line.

* **Cram files** -- Raw sequencing reads are distributed in cram format.
These are distributed by the [EGA](https://www.ebi.ac.uk/ega/studies/EGAS00001001465)
archive for [managed-access]({% post_url 1111-11-11-access %}#managed%}),
cell lines and by the [ENA](http://www.ebi.ac.uk/ena/data/view/ERP007111) archive
for [open-access]({% post_url 1111-11-11-access %}#open) cell lines.
* **VCF files** -- Variant calls in vcf format
are distributed by the [EGA](https://www.ebi.ac.uk/ega/studies/EGAS00001001465)
archive for [managed-access]({% post_url 1111-11-11-access %}#managed%}),
cell lines and by the [EVA](http://www.ebi.ac.uk/eva) archive
for [open-access]({% post_url 1111-11-11-access %}#open) cell lines.

##Mapping

HipSci's exome-seq analysis pipeline comprises these steps:

1. Mapping to the human GRCh37 reference with [BWA](https://github.com/lh3/bwa).
2. Realignment around known indels using GATK
3. Base quality score recalibration using GATK
4. Calculating bq score with samtools

HipSci's FTP site contains:

* [Full command lines](ftp://ftp.hipsci.ebi.ac.uk/vol1/ftp/technical/) used for each step of the exome-seq mapping pipeline
* The [reference genome](ftp://ftp.hipsci.ebi.ac.uk/vol1/ftp/reference/)
* [Known sites and known indels](ftp://ftp.hipsci.ebi.ac.uk/vol1/ftp/reference/) used for realignment and base quality recalibration

##Variant calling

Variants are called from the aligned bam files using samtools mpileupe and bcftools. The full command line is on the [FTP site](ftp://ftp.hipsci.ebi.ac.uk/vol1/ftp/technical/)

The following filters are applied to called variants:

* minimum read depth DP<=4
* maximum read depth DP>2000
* minimum mapping quality MQ<=25
* minimum quality for SNPs QUAL<=30
* minimum quality for indels QUAL<=60
