---
layout: assay
title: RNA-seq
tags:
    - assay
browser_text: Cell lines with RNA-seq data
browser_link: /lines?Assays[]=RNA-seq
---

RNA-sequencing is performed on all HipSci iPS cell lines selected for banking
after passing QC. Sequencing and primary analysis  are performed at the
Wellcome Trust Sanger Institue. The experimental protocols are available on the
[SOP pages]({% post_url 1111-11-11-sop %}).

##Getting the data

The [cell lines and data browser]({{'/lines/#' | append: page.browser_link | prepend: site.baseurl}})
contains links to HipSci's RNA-seq data for any cell line.

* **Cram files** -- Raw sequencing reads are distributed in cram format.
These are distributed by the [EGA](https://www.ebi.ac.uk/ega/studies/EGAS00001001465)
archive for [managed-access]({% post_url 1111-11-11-access %}#managed%}),
cell lines and by the [ENA](http://www.ebi.ac.uk/ena/data/view/ERP007111) archive
for [open-access]({% post_url 1111-11-11-access %}#open) cell lines.
* **Realigned bam files** -- Reads aligned to the GRCh37 reference genome
are distributed by the [EGA](https://www.ebi.ac.uk/ega/studies/EGAS00001001465)
archive for [managed-access]({% post_url 1111-11-11-access %}#managed%}),
cell lines and by the [ENA](http://www.ebi.ac.uk/ena/data/view/ERP007111) archive
for [open-access]({% post_url 1111-11-11-access %}#open) cell lines.


##Mapping

RNA-seq reads are mapped to the GRCh37 reference genome using the [STAR spliced aligner](https://github.com/alexdobin/STAR).
The mapping uses version 19 of the
Gencode gene annotation to enable splice-aware alignments. The reference genome and annotation files can be found
on the HipSci [FTP site](ftp://ftp.hipsci.ebi.ac.uk/vol1/ftp/reference/).

Command line used for generating the genome and junctions index:

    STAR --runMode genomeGenerate \
         --genomeDir ${genomeDir} \
         --genomeFastaFiles ${reference_fasta} \
         --genomeSAindexNbases 14 \
         --runThreadN 16 \
         --limitIObufferSize 150000000 \
         --sjdbGTFfile ${annotation} \
         --sjdbOverhang 99

Command line used for mapping:

    STAR --genomeDir ${genomeDir} \
         --readFilesIn ${fastqs} \
         --outSAMstrandField intronMotif \
         --outSAMunmapped Within \
         --runThreadN 16

