---
layout: page
title: Proteomics assays
permalink: /workflow/proteomics/
---

The HipSci project includes mass spectrometry (MS)-based measurements of protein expression in many of the HipSci cell lines, from both normal and disease groups. This includes both label-free and TMT measurements of protein abundance. Work is on-going to closely integrate the quantitative measurements of protein expression levels with parallel data generated within HipSci that document genome sequences and methylation patterns, mRNA expression and cell phenotypes, measured on the same cell lines.

Our proteomics data set currently spans in total ~17,000 different proteins that we can detect expressed in human iPS cells, together with >480,000 peptides and ~28 million peptide-spectral matches. The protein data from normal cell lines are being used to analyse human genetics and to map protein quantitative trait loci (pQTLs) in iPS cells.

All the resulting HipSci raw MS data are available for download from the [PRIDE](http://www.ebi.ac.uk/pride) repository. In addition, the corresponding processed MS data can be accessed via the [Encyclopedia of Protein Dynamics](https://www.peptracker.com/epd/analytics/?section_id=40100) (EPD), which has a section dedicated to HipSci data. This is a rapidly developing resource and is regularly updated with new data, new interactive features and further integration of the proteome data with other HipSci data sets from both normal and disease cell lines.
