**Pending selection (P):** Cell lines yet to undergo QC, or cell lines awaiting a decision on whether they will be selected and/or banked. Typically, HipSci thrives to derive two iPS cell lines for each donor, both of which will be selected for banking at a cell bank, if they pass QC.

**Not selected (N):** Cell lines that will never be available from a cell bank. Typically, this means another cell line from the same donor has been preferentially selected, or the line has failed to pass QC.

**Selected for banking <span style="color:#FF0004;">(B)</span>:** Cell lines that have passed HipSci's QC standards, and will continue in the pipeline towards being frozen down and banked at EBiSC and/or ECACC

**Banked at ECACC <span style="color:#004ABC;">(B)</span>:**  Cell lines that are currently available from the  [European Collection of Authenticated Cell Cultures](https://www.phe-culturecollections.org.uk/products/celllines/hipsci/index.jsp)

**Banked at ECACC and EBiSC <span style="color:#006C0C">(B)</span>:** Cell lines that are available from the  [European Collection of Authenticated Cell Cultures](https://www.phe-culturecollections.org.uk/products/celllines/hipsci/index.jsp) and [European Bank for induced pluripotent stem cells](http://www.ebisc.org/)

**M (Managed access):** The consent agreement of some HipSci donors authorises release of individually unique data only for specific research use to bona fide researchers. Cell lines on this page are marked as 'Managed access' if the donor's individually unique data are bound by these restrictions.

Managed access data are stored in the [EGA](https://ega-archive.org/) archive. For access to these data, researchers must register and apply for access via WTSI's [Electronic Data Access Mechanism](https://www.sanger.ac.uk/legal/DAA/MasterController)

**O (Open access):** The consent agreement of some HipSci donors authorises release of individually unique data to all parties, with no requirement to satisfy any data access restrictions. Cell lines on this page are marked as 'Open access' if there are no restrictions on the donor's individually unique data.

**Gexarray=Expression array:** Transcription profiling by array is used to measure the activity of genes in the HipSci cell lines. HipSci assays the transcriptome expression profile in all iPS cell lines and in the primary tissue cells (e.g. fibroblasts). HipSci's QC procedure uses transcript expression to assess that derived cell lines have a profile typical of the pluripotent state.

**Gtarray=Genotyping array:** HipSci assays for genotypes in all iPS cell lines and in the primary tissue cells from which they were derived (e.g. fibroblasts). HipSci's QC procedure compares the called genotypes between the primary somatic and derived iPS cells of the same donor, to assess genomic integrity in the derived lines.

**Exomeseq=Exome sequencing:** Exome sequencing is performed on all HipSci iPS cell lines selected for banking after passing QC. We have also generated exome sequencing data for ~250 healthy donor somatic cells. Sequencing and primary analysis are performed at the Wellcome Trust Sanger Institute.

**Rnaseq=RNA sequencing:** RNA-sequencing is performed on all HipSci iPS cell lines selected for banking after passing QC. Sequencing and primary analysis are performed at the Wellcome Trust Sanger Institute.

**Mtarray=Methylation array:** Methylation profiling by array is used to probe the epigenetic state of HipSci iPS cell lines. HipSci assays iPS cell lines that are selected for banking after passing QC. This assay and primary analysis is performed at the Wellcome Trust Sanger Institute.

**Proteomics:** HipSci’s proteomics work is carried out at the University of Dundee. The HipSci project includes mass spectrometry (MS)-based measurements of protein expression in many of the HipSci cell lines, from both normal and disease groups. This includes both label-free and TMT measurements of protein abundance. Work is on-going to closely integrate the quantitative measurements of protein expression levels with parallel data generated within HipSci that document genome sequences and methylation patterns, mRNA expression and cell phenotypes, measured on the same cell lines.

**Cellbiol-fn=Cellular phenotyping:** HipSci’s cellular phenotyping assays are carried out at King’s College London, at the Centre for Stem Cells & Regenerative Medicine. Imaging the HipSci cell lines and using high content analysis, we quantify and standardise phenotypic features that are informative of cell behaviour in an artificial microenvironment. Methods have been developed to facilitate the integration of dynamic and end-point imaging data, with genomics and proteomics data. This allows for the study of genetic contribution on cellular phenotype, and the association of single nucleotide changes to parameters such as cell adhesion, morphology and proliferation.

**Wgs=Whole genome sequencing:** A small selection (~150 donors) of iPS cell lines and their primary somatic cells have been deep whole genome sequenced at 30x coverage.

